### PR TITLE
upgraded code to work with TYPO3 v10

### PR DESCRIPTION
**Pull request by Wipster**

### CODE REVIEW

**Comment by kaystrobach** [comment]
thank you - you can skip the styling as styleci anyway creates a PR if there is a problem with styling. :D

**Comment by Wipster** [comment]
Okay, but it called my attention to a lot of useless format changes probably unnoticed made by the IDE. So I cleaned that up and made a new PR.

File: Classes/Domain/Model/AbstractTheme.php [hunk 1]
```diff
@@ -406,7 +406,7 @@ public function getTypoScriptForLanguage(&$params, &$pObj)
                     $buffer .= ' isoCodeShort = ' . $language['iso-639-1'] . LF;
                     $buffer .= ' isoCodeHtml = ' . $language['hreflang'] . LF;
                     $buffer .= '} ' . LF;
-                    $buffer .= '[end]' . LF;
+                    $buffer .= '[global]' . LF;
```
[the remark this answers is not in the document]
Comment (kaystrobach) [reply Classes/Domain/Model/AbstractTheme.php:409]: @wipster can we stay with end here or is global mandatory?

[the remark this answers is not in the document]
Comment (Wipster) [reply Classes/Domain/Model/AbstractTheme.php:409]: Yes absolutely! It was not planned to create a PR from that code :)

**Comment by kaystrobach** [comment]
@achimfritz can you also take a look?

**Comment by kaystrobach** [comment]
Now we have some code duplicates, but that should be fixable:

* https://app.codacy.com/manual/kaystrobach/themes/pullRequest?bid=21115075&prid=6541597&utm_campaign=new_commit&utm_medium=Email&utm_source=Internal&

**Pull request closed without merging**